Makefile

```diff
@@ -2,10 +2,15 @@ GO := go
 
 GO_BUILD_PACKAGES := ./cmd/...
 GO_BUILD_BINDIR :=./bin
+GIT_COMMIT := $(or $(SOURCE_GIT_COMMIT),$(shell git rev-parse --short HEAD))
```

Comment:
If I build this, this will show the most recent tag instead of a `devel` version which would be expected

Reply:
Ack. Added a `buildData` field to add `+dev` to that version in the Makefile. WDYT?